### DISCUSSION TITLE
[Months] How i can show the default months?

### DISCUSSION THREAD

**Discussion by lariane-guerra** [gpbl/react-day-picker#2276] [Support]
I need to show 2 months. The actual month, for example, July, and the past month, in this case, June.

So:
June July, and not
July August

Its possible?

**Comment by gpbl** [comment]
@lariane-guerra please share some code if you need help. Thanks.

PS. reminder that dates in javascript have months starting from `0` (January):

```tsx
<DayPicker defaultMonth={ new Date(2024, 5) } numberOfMonths={2} />
```

The code above will show June and July.

**Reply by lariane-guerra** [reply]
Yeah, good, this work fine.
But will be natural?
Example:
When we be on August will show:
July August?

**Reply by lariane-guerra** [reply]
Hi brow, i use the function utils subMoth from datefns
`defaultMonth={subMonths(new Date(), 1)}` to be dynamic.

If someone need, its works fine for me.

**Comment by lariane-guerra** [comment]
Hi, how are you today?
I'll try this solution.

return with news.
